include/swift/Runtime/Metadata.h

```diff
@@ -2337,11 +2339,15 @@ struct ClassExistentialContainer {
     return reinterpret_cast<const WitnessTable* const *>(this + 1);
   }
 
-  void copyTypeInto(ClassExistentialContainer *dest, unsigned numTables) const {
+  void copyTypeInto(ClassExistentialContainerImpl *dest,
+                    unsigned numTables) const {
```

Comment:
Minor request (totally unnecessary for 3.0): you could make this templated over the dest's value type, since it doesn't touch that field at all.